Symfony/CS/Tests/Fixer/Symfony/JoinFunctionFixerTest.php

```diff
@@ -1,83 +0,0 @@
-<?php
-
-/*
- * This file is part of the PHP CS utility.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Symfony\CS\Tests\Fixer\Symfony;
-
-use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
-
-/**
- * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
- */
-class JoinFunctionFixerTest extends AbstractFixerTestBase
-{
-    /**
-     * @dataProvider provideCases
-     */
-    public function testFix($expected, $input = null)
-    {
-        $this->makeTest($expected, $input);
-    }
-
-    public function provideCases()
-    {
-        return array(
-            // valid cases
-            array('<?php $foo->join($a);'),
-            array('<?php joinFoo($a);'),
-            array('<?php foo_join($a);'),
-            array('<?php new join($a);'),
-            array('<?php new Foo\join($a);'),
-            array('<?php Foo::join($a);'),
-            array('<?php new join\bar($a);'),
-            array('<?php join::bar($a);'),
-            array('<?php join\bar($a);'),
-            array('<?php \join($a);'),
```

Comment:
This should be fixed. It is the only case where we really know that the global function is called. ;)